src/main/java/practice/StreamPractice.java

```diff
@@ -23,7 +35,11 @@ public int findMinEvenNumber(List<String> numbers) {
      * But before that subtract 1 from each element on an odd position (having the odd index).
      */
     public Double getOddNumsAverage(List<Integer> numbers) {
-        return 0D;
+        return IntStream.range(0, numbers.size())
+                .map(i -> i % 2 == 0 ? numbers.get(i) : numbers.get(i) - 1)
```

Comment:
```suggestion
                .map(isNumberEven ? numbers.get(i) : numbers.get(i) - 1)
```
1 магічне число потрібно додати константу або використати `Math.decrementExact()`
`private static final int DECREMENT = 1;`